### PR TITLE
Support merging Changes

### PR DESCRIPTION
**Pull request by yazad3**
### Description:

Allow merging `Change`s which is in turn a merge of composing `FeatureChange`s.

### Potential Impact:

None.

### Unit Test Approach:

Unit tests added to test various cases of merge of `Change`s

### Test Results:

Test successful.

------

In doubt: [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

**Comment by lucaspcram** [comment]
Great I will take a look at this shortly. I am currently working on improving some of the merging semantics so it's actually good to have some interest in this area.

**Comment by yazad3** [comment]
Thank you @matthieun & @lucaspcram

File: src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeAtlas.java [hunk 1]
```diff
@@ -47,17 +47,17 @@
     private transient Long numberOfPoints;
     private transient Long numberOfRelations;
 
-    public ChangeAtlas(final Atlas source, final Change change)
+    public ChangeAtlas(final Atlas source, final Change... changes)
```
Comment (lucaspcram) [inline src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeAtlas.java]: 👍

**Comment by yazad3** [comment]
@lucaspcram - thank you very much for the merge.

**Pull request merged**